src/utils/keyboardShortcuts.ts

```diff
@@ -23,6 +23,11 @@ const shortcuts = [
     mac: '⌘ + Shift + o',
     text: `Navigate to a symbol in the specification`,
     windows: 'Ctrl + Shift + o'
+  },
+  {
+    mac: '⌘ + 3',
```

Comment:
Why this combination?

Reply:
@domoritz could you suggest any other keyboard shortcut for that? I don't have any specific reason for adding `Cmd+3` as one.

Reply:
Something with e in it?